app/src/path.js

```diff
@@ -4,22 +4,24 @@ export const search = _params => "/search";
 export const feed = _params => "/feed";
 export const projects = _params => "/projects";
 
-export const projectOverview = (params = {}) =>
-  `/projects/${params.id}/overview`;
-export const projectFeed = (params = {}) => `/projects/${params.id}/feed`;
-export const projectMembers = (params = {}) => `/projects/${params.id}/members`;
-export const projectFunds = (params = {}) => `/projects/${params.id}/funds`;
-export const projectSource = (params = {}) => {
-  if (params.revision && params.path) {
-    return `/projects/${params.id}/source/${params.revision}/${params.objectType}${params.path}`;
+export const projectOverview = (domain, name) => `/projects/${domain}/${name}/`;
```

Comment:
Either make this `/projects/${domain}/${name}/overview` or add another sub-route to `ProjectBreadcrumbs.svelte`:
`"/projects/:domain/:name/": Overview,`. Otherwise the overview breadcrumb is broken.